modules/ROOT/pages/apikit-graphql-api-implementation.adoc

```diff
@@ -0,0 +1,525 @@
+}
+----
+
+==== Trigger a Request
+
+----
+curl --location --request POST 'http://localhost:8081/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{"query":"mutation CreatePost($userId:ID!,$title:String!,$body:String!){createPost(userId:$userId,title:$title,body:$body){id user{username}title body comments{body}}}","variables":{"userId":42,"title":"GraphQL by example","body":"Let'\''s talk about GraphQL"}}'
+----
+
+==== Get a Response
+
+----
+{
+   "data": {
+       "createPost": {
+           "id": "101",
+           "user": {
+               "username": "tblackmore15"
+           },
+           "title": "GraphQL by example",
+           "body": "Let's talk about GraphQL",
+           "comments": []
+       }
+   }
+}
+----
+
+=== Add a Comment to the New Post
+
+This use case posts a comment in reply to the post created in the previous step. It also asks for the service to return the body of the comment created and the original post with its list of comments to check that the new one is included.
+
+==== Make a Query
+
+----
+mutation PostComment($userId: ID!, $postId: ID!, $body: String!) {
+   postComment(userId: $userId, postId: $postId, body: $body) {
+       id
+       body
+       post {
+           id
+           title
+           comments {
+               id
+               user {
+                   username
+               }
+               body
+           }
+       }
+   }
+}
+----
+
+==== Make Variables
+
+----
+{
+   "userId": 34,
+   "postId": 101,
+   "body": "Nice post! Thanks for sharing! It's just what I was looking for"
+}
+----
+
+==== Trigger a Request
+
+----
+curl --location --request POST 'http://localhost:8081/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{"query":"mutation PostComment($userId:ID!,$postId:ID!,$body:String!){postComment(userId:$userId,postId:$postId,body:$body){id body post{id title comments{id user{username}body}}}}","variables":{"userId":34,"postId":101,"body":"Nice post! Thanks for sharing! It'\''s just what I was looking for"}}'
+----
+
+==== Get a Response
+
+----
+{
+   "data": {
+       "postComment": {
+           "id": "201",
+           "body": "Nice post! Thanks for sharing! It's just what I was looking for",
+           "post": {
+               "id": "101",
+               "title": "GraphQL by example",
+               "comments": [
+                   {
+                       "id": "201",
+                       "user": {
+                           "username": "ekosex"
+                       },
+                       "body": "Nice post! Thanks for sharing! It's just what I was looking for"
+                   }
+               ]
+           }
+       }
+   }
+}
+----
+
+=== Reply to the New Comment in the Post
+
+This example request adds a comment of the user that created the original post in response to the previous comment. The query asks the service to return all the information of the post in addition to the new reply.
```

Comment:
Can you simplify the first sentence? It's a little hard to parse. Is this just including the comment in the response to the comment?

Reply:
What about `This request example creates a reply in response to the comment posted in the previous step`?

Reply:
I like it! Thanks!